### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added install steps, light formatting changes.

### CODE REVIEW

File: website/site/content/guide/user/_index.md [hunk 1]
```diff
@@ -9,49 +9,103 @@ This guide is an overview of the basic features to get you started. Focalboard i
 
 [Let us know](/feedback/) if there's a feature you are looking for, or [connect with our vibrant community](/contribute/getting-started/) to help bring it to life.
 
+## Enabling Focalboard
```
Comment (chenilim) [inline website/site/content/guide/user/_index.md:12]: Thanks Justine! I've added this section to a [new page under download \ mattermost here](https://github.com/mattermost/focalboard/blob/main/website/site/content/download/mattermost/_index.md). You can remove it from this users guide page.

File: website/site/content/guide/user/_index.md [hunk 2]
```diff
@@ -9,49 +9,103 @@ This guide is an overview of the basic features to get you started. Focalboard i
...
+
+In Mattermost, select the Focalboard icon in the channel header to access boards for that channel.
+
 ## Adding new Boards
 
-1. Click "+ Add Board" at the lower left of the sidebar to add a new board to Focalboard.
-2. Pick "Project Tasks" from the list of templates.
-3. This shows the first view of the new board, which is a table of all tasks
+1. Select **+ Add Board** in the bottom left of the sidebar to add a new board to Focalboard.
+2. Pick **Project Tasks** from the list of templates.
```
Comment (chenilim) [inline website/site/content/guide/user/_index.md:70]: This step should be something like "Pick a template, such as **Project Tasks** from the list".

File: website/site/content/guide/user/_index.md [hunk 3]
```diff
@@ -9,49 +9,103 @@ This guide is an overview of the basic features to get you started. Focalboard i
...
 ![image](./by%20status.png)
+
 5. Click on the properties menu, and turn on "Priority" and "Date created". This will add those properties to the card display.
+
 ![image](./by%20status%20properties.png)
-6. You can likewise change the "Group By", "Filter", and "Sort" settings of the view
-7. In general, all changes you make are saved immediately
+
+6. You can likewise change the **Group By**, **Filter**, and **Sort** settings of the view.
+7. In general, all changes you make are saved immediately.
```
Comment (chenilim) [inline website/site/content/guide/user/_index.md:84]: Suggest "In general, all changes you make are saved immediately, and are visible to all other users".

**Review by chenilim — CHANGES_REQUESTED** [review]
Thanks @justinegeffen! Some minor edits, but looks great!

File: website/site/content/guide/user/_index.md (hunk 2 above)
Comment (justinegeffen) [inline website/site/content/guide/user/_index.md:70]:
```suggestion
2. Pick a template, such as **Project Tasks**, from the list.
```

File: website/site/content/guide/user/_index.md [hunk 4]
```diff
@@ -9,49 +9,103 @@ This guide is an overview of the basic features to get you started. Focalboard i
...
```
Comment (justinegeffen) [inline website/site/content/guide/user/_index.md:84]:
```suggestion
7. In general, all changes you make are saved immediately and are visible to all other users.
```

File: website/site/content/guide/user/_index.md (hunk 1 above)
Comment (justinegeffen) [reply website/site/content/guide/user/_index.md]: Great! I've removed this section in this branch - let me know if that's correct? Then when we merge this the new page you created will be applied and I'll cross-check that it's correct. 

We just need to give Amy the right URL for the release notes - and that will be the new page you created.

**Review by chenilim — APPROVED** [review]
Thanks @justinegeffen! LGTM 👍

**Pull request merged**